src/picterra/__main__.py

```diff
@@ -19,8 +19,19 @@
 
 
 def parse_args(args):
+    # Instantiate client, just for checking URl and api key
+    try:
+        client = APIClient()
+        target = client.base_url
+        suffix = "Your API key starts with '%s'" % client.api_key[:4]
+        desc = "You are targeting %s.\n\n%s" % (target, suffix)
+        del client
+    except APIError:
+        desc = "ATTENTION: you do not have an Api Key: set one before using the cli commands."
```

Comment:
This message won't be printed anywhere and we'll actually not show the underlying APIError. I wouldn't try to be too smart here and just let it fail

Reply:
Checked (got rid of all)